### ISSUE TITLE
Compile failure on new Ruby build

### ISSUE THREAD

**Issue by mbhnyc** [cldwalker/debugger-linecache#6]
Heroku just updated its Ruby stack 1.9.3-p392, and bundle install is now failing :-1: 

Here is the stack trace (from Heroku)

```
Installing debugger-linecache (1.1.2)
       Gem::Installer::ExtensionBuildError: ERROR: Failed to build gem native extension.
       /tmp/build_1ir2dzszqev5i/vendor/ruby-1.9.3/bin/ruby extconf.rb
       *** extconf.rb failed ***
       Could not create Makefile due to some reason, probably lack of
       necessary libraries and/or headers.  Check the mkmf.log file for more
       details.  You may need configuration options.
       Provided configuration options:
       --with-opt-dir
       --without-opt-dir
       --with-opt-include
       --without-opt-include=${opt-dir}/include
       --with-opt-lib
       --without-opt-lib=${opt-dir}/lib
       --with-make-prog
       --without-make-prog
       --srcdir=.
       --curdir
       --ruby=/tmp/build_1ir2dzszqev5i/vendor/ruby-1.9.3/bin/ruby
       /tmp/build_1ir2dzszqev5i/vendor/ruby-1.9.3/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require': cannot load such file -- debugger/ruby_core_source (LoadError)
       from /tmp/build_1ir2dzszqev5i/vendor/ruby-1.9.3/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
       from extconf.rb:2:in `<main>'
       Gem files will remain installed in /tmp/build_1ir2dzszqev5i/vendor/bundle/ruby/1.9.1/gems/debugger-linecache-1.1.2 for inspection.
       Results logged to /tmp/build_1ir2dzszqev5i/vendor/bundle/ruby/1.9.1/gems/debugger-linecache-1.1.2/ext/trace_nums/gem_make.out
       An error occurred while installing debugger-linecache (1.1.2), and Bundler cannot continue.
       Make sure that `gem install debugger-linecache -v '1.1.2'` succeeds before bundling.
```

**Comment by stepheneb** [comment]
debugger-ruby_core_source has been updated to v1.1.9 which includes source for 1.9.3-p392 ... but the gem isn't appearing yet on rubygems.org, see: cldwalker/debugger-ruby_core_source#17

However after updating my Gemfile to reference the git repo for debugger-ruby_core_source

```
gem "debugger-ruby_core_source", :git => "git://github.com/cldwalker/debugger-ruby_core_source.git"
gem "debugger",    "~> 1.3.1"
```

I'm still getting the same error listed above trying to install  debugger-linecache (1.1.2).

**Comment by cldwalker** [comment]
Released 1.3.2. Should fix this

**Comment by JAStanton** [comment]
`bundle update debugger` is what fixed it for me
